functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -108,7 +109,103 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 			Expect(cInfo).To(BeEquivalentTo(expectedInfo), "regression test failed for '%s' case", expectedClusterInfoPath)
 		}
 	})
-
+	Context("Systems with Hyperthreading enabled", func() {
+		It("[test_id:41419] Verify PPC script fails when reserved cpu count is 2 and requires to split across numa nodes", func() {
+			Expect(ppcPath).To(BeAnExistingFile())
+			mustGatherFullPath := path.Join(mustGatherPath, "must-gather.bare-metal")
+			Expect(mustGatherFullPath).To(BeADirectory())
+			cmdArgs := []string{
+				fmt.Sprintf("--disable-ht=%t", false),
+				fmt.Sprintf("--mcp-name=%s", "worker-cnf"),
+				fmt.Sprintf("--must-gather-dir-path=%s", mustGatherFullPath),
+				fmt.Sprintf("--reserved-cpu-count=%d", 2),
+				fmt.Sprintf("--rt-kernel=%t", true),
+				fmt.Sprintf("--split-reserved-cpus-across-numa=%t", true),
+				fmt.Sprintf("--user-level-networking=%t", false),
+				fmt.Sprintf("--profile-name=%s", "Performance"),
+			}
+			_, err := testutils.ExecAndLogCommand(ppcPath, cmdArgs...)
+			ppcErrorString := errorStringParser(err)
+			Expect(ppcErrorString).To(ContainSubstring("failed to compute the reserved and isolated CPUs: can't allocate odd number of CPUs from a NUMA Node"))
```

Comment:
this is fragile. Do we have a better way? Do we want to consider error message part of the API?
Probably too late to file a RFE for this, though.